ai21/clients/common/completion_base.py

```diff
@@ -25,6 +25,7 @@ def create(
         presence_penalty: Optional[Penalty] = None,
         count_penalty: Optional[Penalty] = None,
         epoch: Optional[int] = None,
+        logit_bias: Optional[Dict[str, float]] = None,
```

Comment:
In my pr: https://github.com/AI21Labs/ai21-python/pull/69 , I'm changing the approach for the default parameters for function calling:
instead if `Optional[Type] = None` We'll have `Type | NotGiven = NOT_GIVEN`.
`NOT_GIVEN` is essentially the equivalent typescript's `undefined`, and instead of sending `null` in the request to the backend, we are not sending the field in the body if it's `NOT_GIVEN`.
Please adjust accordingly